modules/ssh-keys.nix

```diff
@@ -1,144 +1,96 @@
 {
-  artemohanjanyan = [
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQMCTlz9c84peYN7xXp1NWmJP0mCYLxnw8llWw9FS1EQsdJZeTmJesX1It8dTQ4khAxCtBpAacWphgZjjQquz9KmnSp245Dft4JYVCoHRrwez9DCNn4RwkfmnJTYLHt9vB77yMSwluoKFD54KjTBaLlgdQ9XPdFt7xWkpdYPWbTmuEx7bEpLMgwnHVrZ24i+FxEaIbnkJXVRzMB4MKdoCzJauqGUHtvXDVRdxU1kw93x0i9fxr2XjZK76I98z8Uz2WiMBZwaXsFZ8XYiSNOomwsIjHCkH+657D11U0A62euVffogr8XM9q51zDyvhREcWFtdkKKRSSlQIB8uHqicW7"
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDdJlxUtDGFoEEtsXKxl7aR2CKso+rVNDK2oPqlENJgvspJpAdJ9yW3GpD3yDT0YK2klhitk4jH9bov1Xg76nbw3/VvF63M1s+PWlhZyzeQ+FkOwnJD7+9/UuxrK7W0OYtcUiG+/2kbpyMBcyTXvYhd7pXH0O5A3UXRCmcpFBZw0+lZ+NoY9NdRVctmJCYzXl8ZVr/gL3sghrudBWE0gvnYggRmAyPzVW2+dV7VcUw/IaDba1AaA9VI/FXiYPLCo7jFg1Yu7wEGh7K0dNI4YbelqsDayVVauJwhPBr3YpEcU1ASIVUegVU93GFT35Sv4VPNSA8xh8lTzozpCtdoj0pl"
-  ];
-  awson = [
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCwV8tam8peOJ50gnbYeRhT6PBC9tFXYdUWG0pkVBcCBp4KpP+W5MOzsYC7Q+9NJpn61Fg9rT+6CQpk+HTjlgeFu8IadJ55K/ToAd2mXFNRDk/lHMfvK8+epg0Wwnf8oQkIB8S/DfzQ9S2jlkBQhSpeRYk4KkhvZgDKQDrW0lzicJI0kVHnTm3s1yvhDB6jDXMY6TGUtk2L82pryqxiOpYOeOydPGrBlcBjxkZY4iQIjoqnNJxsH/cwl0W9JszglR90gVXH6q1qirqHeDmKwQoB1FILmblf2GNh+MwJNJM6J64s8DgDL9ChkEDGfuJ5UFZZrB114p1vTqlW2fB0zsMt"
-  ];
-  ayrat555 = [
-    "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC0oUFeB01KvPAzV+lSjLh+0oRCJULmEvYQP8usFofXUKquWqEaC5XW9Kb1O0Fr5XcARs2GSofsQ83QKgYLYJjT76dcCULrRfHW1acF5bmdHLce14hSVmAedOnGWKlPjQwe/N3JaHV1LgGQM541ZwJDBBdyExmkTnFsFJqZfACrvOCxWnJFbl71xNtHRz1kSiqQOfucTwxdxxFdNSzxZv+l0ed2pd05RW1susyDYJd5whsZ+DsIkj46X2fyFVR9BUSujn6UGdqD2DvlG45Dyk0VonDJu5CKXTmmv2HwDpxSlMLsBGaWnn1L7NKDwS9UjUHuaKpDO90ZYFi97PHh5OqViJ2BmrfmlVIzAjoPFJywWbC3c0hSw1moFLafhaiucBJ2YV+LwlZvL+3UWAgmfH9eASPnmMIbb/J9pW9Ub1eIRbiAv/Zg/4QApAASi7kXLwtCpvCHmem+v69ioFFzvs3lf/fzM/oKjLPejnWi3ESFbTx+sC5Jo1vTDzIzLYeTAU/eUDNtmXsQ8xa2BhGoH9qsmA5Wju/NklO1YcgKHJTKGwNEw2Cg6dgvKGXJMP06DO+/IlgpV7PZUByPxz9VIa0jjsaugxkM1eeaIBQfyOmhAGQbh0owalEi6eR7Ik5tEuMA3HdmIWzAAPxSl2bFBu662s1KaSJbYy86rds3NzNnmw== ayratin555@gmail.com"
-  ];
+  # Alexander Bantyev
   balsoft = [
     "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDd2OdcSHUsgezuV+cpFqk9+Svtup6PxIolv1zokVZdqvS8qxLsA/rwYmQgTnuq4/zK/GIxcUCH4OxYlW6Or4M4G7qrDKcLAUrRPWkectqEooWRflZXkfHduMJhzeOAsBdMfYZQ9024GwKr/4yriw2BGa8GbbAnQxiSeTipzvXHoXuRME+/2GsMFAfHFvxzXRG7dNOiLtLaXEjUPUTcw/fffKy55kHtWxMkEvvcdyR53/24fmO3kLVpEuoI+Mp1XFtX3DvRM9ulgfwZUn8/CLhwSLwWX4Xf9iuzVi5vJOJtMOktQj/MwGk4tY/NPe+sIk+nAUKSdVf0y9k9JrJT98S/"
   ];
-  builder = [
```

Comment:
Do you know where this `builder` key was used?

Reply:
Looks like some outdated entity that was used with the previous infra configuration